src/components/Confirm/Previews.js

```diff
@@ -8,6 +8,32 @@ import Error from '../Error'
 import theme from '../Theme/style.scss'
 import style from './style.scss'
 
+const getMessageKey = ({
+  capture,
+  documentType,
+  error,
+  forceRetake,
+  method,
+}) => {
+  if (method === 'face') {
+    return `confirm.face.${capture.variant}.message`
+  }
+
+  if (error && error.type === 'error') {
+    return `confirm.${documentType}.message`
+  }
```

Comment:
Wouldn't this if statement be redundant since this method already returns `confirm.${documentType}.message` by default anyway (line 34)?

Reply:
Sorry, I forgot to explain this in place. With this improvement, we have 2 new copies indicating 2 different situations:
* The 1st & 2nd time that user is submitting documents with warnings, using copy at `confirm.document_image_poor.message` with `forceRetake` condition.
* The 3rd time that user is submitting documents with warnings, using copy at `confirm.document_image_medium.message` with `error && error.type === 'warn'` condition

The point is when there's an error and `error.type === 'error'`, it could jump on `forceRetake` case. Then we need an extra `confirm.${documentType}.message` here just to be displayed in case of error.